modules/Formal Assessment/externalAssessment_details.php

```diff
@@ -93,7 +93,7 @@
             externalAssessmentDetails($guid, $gibbonPersonID, $connection2, '', $manage, $search, $allStudents);
 
             //Set sidebar
-            $_SESSION[$guid]['sidebarExtra'] = getUserPhoto($guid, $row['image_240'], 240);
+            $session->get('sidebarExtra') = getUserPhoto($guid, $row['image_240'], 240);
```

Comment:
Hi Harry, this has an equals sign, so it would be a $session->set() method call, rather than get().